sycl/test/lit.cfg.py

```diff
@@ -119,11 +120,11 @@ def getDeviceCount(device_type):
 if getDeviceCount("cpu")[0]:
     found_at_least_one_device = True
     lit_config.note("Found available CPU device")
-    cpu_run_substitute = "env SYCL_DEVICE_TYPE=CPU "
+    cpu_run_substitute = "env SYCL_DEVICE_TYPE=CPU SYCL_BE={SYCL_BE} ".format(SYCL_BE=backend)
```

Comment:
On a fail LIT prints out this information and eases running the test by hand. Though the user still needs to add the `LD_LIBRARY_PATH` to find the SYCL library, but that is not part of this change.